sdk/deviceupdate/azure-iot-deviceupdate/azure/iot/deviceupdate/aio/operations/_operations.py

```diff
@@ -0,0 +1,4026 @@
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+import functools
+from json import loads as _loads
+from typing import Any, AsyncIterable, Callable, Dict, Generic, List, Optional, TypeVar, Union
+import warnings
+
+from azure.core.async_paging import AsyncItemPaged, AsyncList
+from azure.core.exceptions import ClientAuthenticationError, HttpResponseError, ResourceExistsError, ResourceNotFoundError, map_error
+from azure.core.pipeline import PipelineResponse
+from azure.core.pipeline.transport import AsyncHttpResponse
+from azure.core.polling import AsyncLROPoller, AsyncNoPolling, AsyncPollingMethod
+from azure.core.polling.async_base_polling import AsyncLROBasePolling
+from azure.core.rest import HttpRequest
+from azure.core.tracing.decorator import distributed_trace
+from azure.core.tracing.decorator_async import distributed_trace_async
+
+from ...operations._operations import build_management_collect_logs_request, build_management_create_or_update_deployment_request, build_management_create_or_update_group_request, build_management_delete_deployment_request, build_management_delete_group_request, build_management_get_deployment_request, build_management_get_deployment_status_request, build_management_get_device_class_request, build_management_get_device_module_request, build_management_get_device_request, build_management_get_device_tag_request, build_management_get_group_request, build_management_get_group_update_compliance_request, build_management_get_log_collection_operation_detailed_status_request, build_management_get_log_collection_operation_request, build_management_get_operation_request, build_management_get_update_compliance_request, build_management_import_devices_request_initial, build_management_list_best_updates_for_group_request, build_management_list_deployment_devices_request, build_management_list_deployments_for_group_request, build_management_list_device_classes_request, build_management_list_device_tags_request, build_management_list_devices_request, build_management_list_groups_request, build_management_list_installable_updates_for_device_class_request, build_management_list_log_collection_operations_request, build_management_list_operations_request, build_management_retry_deployment_request, build_management_stop_deployment_request, build_updates_delete_update_request_initial, build_updates_get_file_request, build_updates_get_operation_request, build_updates_get_update_request, build_updates_import_update_request_initial, build_updates_list_files_request, build_updates_list_names_request, build_updates_list_operations_request, build_updates_list_providers_request, build_updates_list_updates_request, build_updates_list_versions_request
+T = TypeVar('T')
+JSONType = Any
+ClsType = Optional[Callable[[PipelineResponse[HttpRequest, AsyncHttpResponse], T, Dict[str, Any]], Any]]
+
+class UpdatesOperations:
```

Comment:
Will keep API-related comments to APIview.

Reply:
Python Apiview is still invalid now. And we are waiting for expert to fix. The code has been updated with latest swagger.
![image](https://user-images.githubusercontent.com/70930885/149865390-d8845e30-3b6a-4ce9-b23a-302e12c0c3b3.png)

Reply:
@annatisch Some of your api-view suggest to change operation name. It may be more clear for customers and we could do it with `directive` in `readme.md`. But in view of that it is defined in swagger, I think it is better to keep name same with swagger definition. In the future, the maintenance will be transferred to service team. So I think we had better try to configure as less as possible in `readme.md` now.

Reply:
Agreed that we should try and get as many name changes into the swagger spec as possible - though in a couple of cases the Python SDK may need different names to other languages - in which case directives may be necessary.